### ISSUE TITLE
QR_Code generator

### ISSUE THREAD

**Issue by Apoorvgarg-creator** [python-geeks/Automation-scripts#369]
## Script Title -

**Brief** -
The script will change url, contact card, email , sms , pdf and many more into QR code. 

## Instructions

- Create a new folder for your script and file/folder name should be appropriate.
- Create a README.md in your folder for program Instructions
- add requirements.txt if needed

### Programming Language

- [x] Python

May I solve this as an issue and work on it to add this new feature?

@pawangeek 
Happy Coding

**Comment by pawangeek** [comment]
Sure @Apoorvgarg-creator go ahead, keep that folder name 'qrcode_generator'

**Comment by mharyam** [comment]
Hello, are all the tasks still open?

**Comment by CodeAlpha7** [comment]
Can I help @Apoorvgarg-creator with this issue?
Thanks.

**Comment by Apoorvgarg-creator** [comment]
@mharyam @CodeAlpha7, Yes you can help. I was wondering what will be a better way to convert an image, pdf into QR. Either upload them somewhere on the cloud and then converting the URL into QR-code 
**OR**
Convert them into some encoding (for ex. BASE 64) and convert the string into QR-code and on Scanning the QR-code . decode it into its original format.

**Comment by CodeAlpha7** [comment]
@pawangeek I have completed the code for this. Before making the PR, went through the guidelines and saw that I couldn't make a one unless I'm assigned to the issue. I have discussed with @Apoorvgarg-creator. 
Could I be assigned to this issue so I could make the PR?

Thanks and Regards

**Comment by Apoorvgarg-creator** [comment]
@CodeAlpha7 I have unassigned myself from the task. Now you can make the PR.
